tests/acceptance/features/apiOcm/share.feature

```diff
@@ -1229,3 +1229,81 @@ Feature: an user shares resources using ScienceMesh application
         }
       }
       """
+
+  @issue-10689
+  Scenario: federation user lists ocm shares
+    Given using server "REMOTE"
+    And "Brian" has created the federation share invitation
+    And using server "LOCAL"
+    And "Alice" has accepted invitation
+    And user "Alice" has created folder "folderToShare"
+    And user "Alice" has sent the following resource share invitation to federated user:
+      | resource        | folderToShare |
+      | space           | Personal      |
+      | sharee          | Brian         |
+      | shareType       | user          |
+      | permissionsRole | Editor        |
+    And using server "REMOTE"
+    When user "Brian" lists all available spaces via the Graph API
+    Then the HTTP status code should be "200"
+    And the JSON response should contain space called "Brian Murphy" and match
```

Comment:
we do not have to check the personal space. We can just check that the shares doesn't exist in the list

Reply:
Waiting for PR: https://github.com/owncloud/ocis/pull/10672 to be merged.